JavaScript/JavaScriptSDK/Util.ts

```diff
@@ -148,5 +148,14 @@
                 && columnNumber == 0
                 && error == null;
         }
+
+        /**
+        * Returns string representation of an object suitable for diagnostics logging.
+        */
+        public static dump(object: any): string {
+            var objectTypeDump: string = Object.prototype.toString.call(object);
+            var propertyValueDump: string = JSON.stringify(object);
```

Comment:
I think in this case it makes sense to trace the object size and shrink it. And maybe indicate that the object was cut off to make it easier for later telemetry analysis of how many large errors are there.